includes/definitions/discovery/rutos-2xx.yaml

```diff
@@ -0,0 +1,102 @@
+mib: TELTONIKA-MIB
+modules:
+  sensors:
+    pre-cache:
+      data:
+        - oid:
+            - Operator
+    temperature:
+      data:
+        - oid: Temperature
+          value: Temperature
```

Comment:
You don't need value if oid is the same.

Reply:
I deleted them.